test/models/model.js

```diff
@@ -98,7 +99,7 @@ describe('Model Tests', function(){
             assert.throws( function() {resource.setPropertyValue('booleanProperty', 'Foo');}, /.+expected type Boolean/, 'did not throw with expected message');
 
             // o DateTime dateTimeProperty
-            const dateTime = new Date('2016-10-11T02:30:26.571Z');
+            const dateTime = Moment.parseZone('2016-10-11T02:30:26Z');
```

Comment:
About size: there is a significant size difference in the bundle with `moment`. Switching to `moment-mini` shows a more reasonable increase in bundle size, from the initial:
```$ npm run prepublishOnly

> composer-concerto@0.60.0 prepublishOnly /Users/dselman/dev/composer-concerto
> webpack --config webpack.config.js --mode production

Hash: 6dabe6228099cceb4565
Version: webpack 4.20.2
Time: 620ms
Built at: 2019-03-18 12:59:24
      Asset     Size  Chunks             Chunk Names
concerto.js  236 KiB       0  [emitted]  client
```
to
```
$ npm run prepublishOnly  

> composer-concerto@0.60.0 prepublishOnly /Users/jeromesimeon/git/composer-concerto
> webpack --config webpack.config.js --mode production

Hash: f1ceb64cdc8126b192c3
Version: webpack 4.20.2
Time: 7626ms
Built at: 2019-03-18 10:51:05
      Asset     Size  Chunks                    Chunk Names
concerto.js  293 KiB       0  [emitted]  [big]  client
```

Reply:
I forgot to ask about milliseconds. This is the default behavior for parsing/formatting in moment.js. This could be adjusted to preserves milliseconds if important.